CHANGELOG.md

```diff
@@ -7,4 +7,10 @@
 ### Fixed
 1. [#12] Output function doesn't accept parameter 'module'
 1. [#16] Handle empty space/special characters when passing string to command line options
-1. Tested with terraform 0.10.0 
\ No newline at end of file
+1. Tested with terraform 0.10.0
+
+## [0.10.0]
+### Fixed
+1. [#27] No interaction for apply function
+1. [#18] Return access to the subprocess so output can be handled as desired
+1. [#24] Full support for output(); support for raise_on_error
```

Comment:
that's very useful, hope it can be released soon

Reply:
@giorgiosironi sorry for the delay, this has been released now.

Reply:
Thank you, I hope to contribute back to this project